garrysmod/gamemodes/terrortown/gamemode/shared.lua

```diff
@@ -139,9 +139,9 @@ local ttt_playercolors = {
    }
 };
 
-CreateConVar("ttt_playercolor_mode", "1")
+local playercolor_mode = CreateConVar("ttt_playercolor_mode", "1")
 function GM:TTTPlayerColor(model)
-   local mode = GetConVarNumber("ttt_playercolor_mode") or 0
+   local mode = playercolor_mode:GetInt() or 0

```

Comment:
I don't think the `or 0` is needed anymore since `GetInt` already returns `0` on failure